### DISCUSSION TITLE
Nobody is helping me to build a success desktop client application

### DISCUSSION THREAD

**Discussion by mirtaqi07** [toeverything/AFFiNE#6667] [Q&A]
Greetings, 

I'm stuck here for days. I've opened the issues, but nobody is responding there also. The `yarn make` built the AFFiNE-canary.exe but on opening it, I get the following error. Although, `yarn run dev:electron` was working well. 

Here it says, about `couldn't find electron-updater` but it is in the node_modules directory. Kindly help to fix it, I'd really appreciate that.

![1](https://github.com/toeverything/AFFiNE/assets/88597955/34d29768-e698-4e17-bbb4-74cb260ed472)

**Comment by pengx17** [comment]
I have closed the other issue you created at https://github.com/toeverything/AFFiNE/issues/6645.
The issue here might because you may not correctly build the package with yarn hoisting OFF.

**Reply by mirtaqi07** [reply]
Kindly see that. I've followed strictly those steps but when opening software it's kind of stuck. [#6645](https://github.com/toeverything/AFFiNE/issues/6645#issuecomment-2075289558)

**Reply by mirtaqi07** [reply]
Dear bro??

**Reply by pengx17** [reply]
See my updated docs in https://github.com/toeverything/AFFiNE/blob/05-01-docs_update_outdated_docs/docs/building-desktop-client-app.md
